addon/initializers/showdown-extension.js

```diff
@@ -89,27 +111,11 @@ export function initialize(/* application */) {
           codeblock = `<div class="filename ${language}"><div class="ribbon"></div><span>${attributes['data-filename'] || ''}</span>${codeblock}</div>`;
         }
       } else {
+        // (bug?) This clause may never be reached as the assert above fires in this case?
```

Comment:
I'm pretty sure that you can remove this comment 🤔 I don't know how this could be a bug

Reply:
that type of assert only happens in dev mode so when you're running in production you can still reach that since the assertion is essentially stripped from a build